app/controllers/finders_controller.rb

```diff
@@ -37,4 +38,8 @@ def facet_params
   def keywords
     params[:keywords] unless params[:keywords].blank?
   end
+
+  def atom_url
+    request.original_url.gsub('.json', '').gsub(finder_slug, "#{finder_slug}.atom")
```

Comment:
@evilstreak: from [this json](https://www.gov.uk/government/announcements.json?keywords=&announcement_filter_option=government-responses&topics%5B%5D=all&departments%5B%5D=all&world_locations%5B%5D=all&from_date=&to_date=&_=1424274252289) which is used to power the annoncements filter:

```json
{
  ...
  "next_page":2,
  "next_page_url":"...",
  "atom_feed_url":"https://www.gov.uk/government/announcements.atom?announcement_filter_option=government-responses",
  "email_signup_url":"..."
}
```